src/sketch.js

```diff
@@ -1262,6 +1262,7 @@ VectorPaintEditorMorph.prototype.populatePropertiesMenu = function () {
         pc = this.propertiesControls,
         alpen = new AlignmentMorph("row", this.padding),
         alignColor = new AlignmentMorph("row", this.padding);
+        alignNames = new AlignmentMorph("row", this.padding);
```

Comment:
excellent! That's what I would've done :-)
Thanks!

Reply:
The semicolon at the end of the `alignColor` line should be a comma, I believe :)

Reply:
oh yeah. Please always test and JSHint/Lint your code.

Reply:
oops.  I tested it a lot but I never think about linting it, because (unlike the original C lint) 3/4 of what it says is arbitrary standards of taste.  :-(  I'll try to remember.

Reply:
That's certainly true of Crockford's JSLint, which has become totally unusable because it forces Doug's personal taste on everybody. You'll find JSHint (https://jshint.com) much more tolerant and usable.